ontology/yaml/resources/fields/telemetry_fields.yaml

```diff
@@ -1079,3 +1079,30 @@ literals:
 - zone_air_no2_probability_percentage_sensor
 - zone_air_no2_ozone_concentration_sensor
 
+## New fields for electrical system additions
+- voltage_sensor
```

Comment:
this feels too vague -- however, in the context of a battery, it really isn't a "line_voltage"

are there any descriptors we could add here that would make sense? charge_voltage_sensor? battery_voltage_sensor feels too specific

Reply:
think this is probably good enough, given that for a standard battery it should be obvious the voltage is across the terminals.